borsh-derive/src/internals/schema/struct/mod.rs

```diff
@@ -2,13 +2,14 @@ use proc_macro2::TokenStream as TokenStream2;
 use quote::{quote, ToTokens};
 use syn::{ExprPath, Field, Fields, Ident, ItemStruct, Path, Type, WhereClause};
 
-use crate::{
-    attribute_helpers::{
+use crate::internals::{
+    attributes::field::schema::ParameterOverride,
+    attributes::{
         contains_skip,
-        field::{self, schema},
+        field::{self},
     },
     generics::{compute_predicates, without_defaults, FindTyParams},
-    schema_helpers::declaration,
+    schema::declaration,
 };
 
```

Comment:
move 
1. `fn visit_field(field: &Field, visitor: &mut FindTyParams)`
2. `fn visit_struct_fields(fields: &Fields, visitor: &mut FindTyParams)`
3. `fn visit_struct_fields_unconditional(fields: &Fields, visitor: &mut FindTyParams)`
up one level to `borsh-derive/src/internals/schema/mod.rs`, as they're used commonly between `structs` and `enums` submodules